<launch>
	<rosparam command="load" file="$(find laserp)/config/default.yaml"/>
	<arg name="respawn" default="false" />
	<arg name="debug" default="false" />
	<arg name="node_name" default="pylon_camera_node" />
	<arg name="config_file" default="$(find pylon_camera)/config/default.yaml" />

	<arg unless="$(arg debug)" name="launch_prefix" value="" />
	<arg     if="$(arg debug)" name="launch_prefix" value="gdb -ex run --args" />

	<node name="$(arg node_name)" pkg="pylon_camera" type="pylon_camera_node" output="screen"
	  respawn="$(arg respawn)" launch-prefix="$(arg launch_prefix)">
	<rosparam command="load" file="$(arg config_file)" />
	</node>
	<node name="laserp_node_name" pkg="laserp" type="laserp_node"/>
</launch>
